aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/customize/DisabledAuthDecorator.kt

```diff
@@ -46,3 +44,5 @@ class DisabledAuthDecorator : ClientCodegenDecorator {
         }
     }
 }
+
+private fun String.shapeId() = ShapeId.from(this)
```

Comment:
This seems like something that we want to reuse throughout the codebase. Can we put it in e.g. `Smithy.kt`?